app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
Code Efficiency: This line accesses class attribute via an instance. It's better to use the class name to get access to the class attributes. So, you should replace `self.alive` with `Animal.alive`.